jekyll/_cci2/configuration-reference.md

```diff
@@ -27,12 +27,62 @@ You can see a complete `config.yml` in our [full example](#full-example).
 
 Key | Required | Type | Description
 ----|-----------|------|------------
-version | Y | String | `2`, `2.0`, or `2.1` See the Reusing Config doc for an overview of new 2.1 keys available to simplify your `.circleci/config.yml` file, reuse, and parameterized jobs.
+version | Y | String | `2`, `2.0`, or `2.1` See the [Reusing Config]({{ site.baseurl }}/2.0/reusing-config/) doc for an overview of new 2.1 keys available to simplify your `.circleci/config.yml` file, reuse, and parameterized jobs.
```

Comment:
We should consider a "What's new in 2.1" doc or a "Config features by version" perhaps rather than consider "Reusing Config" to be synonymous with 2.1, but we can cover that in a later PR.

Reply:
Especially because 2.1 config intentional does not support all the config exceptions that are out there (like Picard syntax)

Reply:
What is the change you want to see in this document?

Reply:
If it's possible to name the link `Reusing Config with Orbs` or similar, I'd go with that. Otherwise this can stay.